pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools", "wheel", "numpy"]
+requires = ["setuptools", "setuptools_scm", "wheel", "numpy>=1.13.0"]
```

Comment:
This needs to be `==` we always need to build the C extension with the oldest version we support.